src/lib/shared/components/ui/FormElements/FormRenderer/FormRenderer.stories.tsx

```diff
@@ -269,6 +273,16 @@ const config: FormConfig<z.infer<typeof schema>> = {
                     label: 'I agree to the terms and conditions',
                     statePath: 'iAgree',
                 },
+                {
+                    id: 'radio',
+                    required: false,
+                    type: 'radio-select',
+                    label: 'Which of these numbers is your favorite?',
+                    helperText: 'Select an option',
+                    fullWidth: true,
+                    statePath: 'radio',
```

Comment:
```suggestion
                    statePath: 'favoriteNumber',
```

Same thing to make this work with the previous comment

Reply:
I did it